tests/test_check.py

```diff
@@ -72,7 +72,7 @@ def test_check_passing_distribution(monkeypatch):
     )
     monkeypatch.setattr(check, "_WarningStream", lambda: warning_stream)
 
-    assert not check.check("dist/*", output_stream=output_stream)
+    assert not check.check(["dist/*"], output_stream=output_stream)
```

Comment:
What changed such that we used to accept a string or a list of strings and now we only accept a list of strings? This seems backwards incompatible to me.

Reply:
I think this has always been an incorrect use of `check`:

https://github.com/pypa/twine/blob/7111240b04f022d5366af7297b9e81c63d76fb6d/twine/commands/check.py#L111-L112

https://github.com/pypa/twine/blob/7111240b04f022d5366af7297b9e81c63d76fb6d/twine/commands/__init__.py#L33-L35

I think it only worked in the tests because `_find_dists` is always stubbed:

https://github.com/pypa/twine/blob/7111240b04f022d5366af7297b9e81c63d76fb6d/tests/test_check.py#L49-L51